### PR TITLE
Feature/bank microservice

### PR DESCRIPTION
**Pull request by MalinavyBusel**

### CODE REVIEW

File: packages/api-gateway/src/app.controller.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
+import { AppService } from './app.service';
+
+@Controller()
+export class AppController {
+  constructor(private readonly appService: AppService) {}
+
+  @Get()
+  getHello(): string {
```
Comment (olegschekotihin) [inline packages/api-gateway/src/app.controller.ts]: unused endpoint

File: packages/api-gateway/src/app.service.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class AppService {
+  getHello(): string {
```
Comment (olegschekotihin) [inline packages/api-gateway/src/app.service.ts]: unused service

File: packages/api-gateway/src/controller/account/account.controller.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  @ApiOperation({
+    summary: 'Returns account',
+    description: 'Returns account with the same UUID',
+  })
+  @ApiParam({
+    name: 'id',
+    description: 'the string representation of the target account UUID',
+  })
+  @ApiOkResponse({ type: ResponseAccountDto })
+  @Get(':id')
```
Comment (olegschekotihin) [inline packages/api-gateway/src/controller/account/account.controller.ts]: this must be first, e.g.
@Get(':id')
@ApiOperation({
    summary: 'Returns account',
    description: 'Returns account with the same UUID',
  })
    @ApiParam({
    name: 'id',
    description: 'the string representation of the target account UUID',
  })
  @ApiOkResponse({ type: ResponseAccountDto })

File: packages/api-gateway/src/controller/account/account.controller.ts [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  ApiOperation,
+  ApiParam,
+  ApiTags,
+} from '@nestjs/swagger';
+
+@ApiTags('Account API')
+@Controller('account')
+export class AccountController {
+  @ApiOperation({
+    summary: 'Returns account',
```
Comment (olegschekotihin) [inline packages/api-gateway/src/controller/account/account.controller.ts]: summary: 'Get account by Id' or 'Find account by Id'

File: packages/api-gateway/src/controller/account/account.controller.ts [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    description: 'Returns account with the same UUID',
```
Comment (olegschekotihin) [inline packages/api-gateway/src/controller/account/account.controller.ts:25]: description may be equal to the summary

File: packages/api-gateway/src/controller/account/dto/response.account.dto.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { Account, WithId, currencyTypesEnum } from 'common-model';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ResponseAccountDto implements Account, WithId {
+  @ApiProperty()
```
Comment (olegschekotihin) [inline packages/api-gateway/src/controller/account/dto/response.account.dto.ts]: Where description?)

File: packages/api-gateway/src/app.controller.ts (hunk 1 above)
Comment (MalinavyBusel) [reply packages/api-gateway/src/app.controller.ts]: убрал

File: packages/api-gateway/src/app.service.ts (hunk 2 above)
Comment (MalinavyBusel) [reply packages/api-gateway/src/app.service.ts]: убрал

File: packages/api-gateway/src/controller/account/account.controller.ts (hunk 3 above)
Comment (MalinavyBusel) [reply packages/api-gateway/src/controller/account/account.controller.ts]: +

File: packages/api-gateway/src/controller/account/account.controller.ts (hunk 4 above)
Comment (MalinavyBusel) [reply packages/api-gateway/src/controller/account/account.controller.ts]: +

File: packages/api-gateway/src/controller/account/dto/response.account.dto.ts (hunk 6 above)
Comment (MalinavyBusel) [reply packages/api-gateway/src/controller/account/dto/response.account.dto.ts]: added

**Pull request merged**